packages/mds-repository/connection-manager.ts

```diff
@@ -78,8 +78,11 @@ export const ConnectionManager = (prefix: string, options: Omit<ConnectionManage
 
   const connect = async (mode: ConnectionMode) => {
     if (!connections) {
-      /* istanbul ignore next */
-      throw RepositoryError.create(Error('Connection manager not initialized'))
+      await initialize()
+      if (!connections) {
```

Comment:
Shouldn't this always be true, since you can't get here unless it's true? Or is connections mutated in initialize?

Reply:
Yes, `initialize` sets `connections`